pudb/remote.py

```diff
@@ -84,6 +84,30 @@
 """
 
 
+class TelnetCharacters:
```

Comment:
I put the characters into this class to group them like they were in telnetlib. I could just copy them into the function that uses them or group them a different way.